src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -94,6 +100,10 @@ def legacy_mode(self):
     def legacy_mode(self, mode):
         self.legacy = mode
 
+    @property
+    def features(self):
+        raise VolumeException(-errno.EINVAL, "invalid to invoke base class method")
```

Comment:
Raising `VolumeException` here might lead to the error string showing up in console (as cmd output). Probably just raise `NotImplementedError `?

Reply:
Addressed in the latest push